bofire/strategies/doe_strategy.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+from pydantic.types import PositiveInt
+
+import bofire.data_models.strategies.api as data_models
+from bofire.strategies.doe.design import find_local_max_ipopt
+from bofire.strategies.strategy import Strategy
+
+
+class DoEStrategy(Strategy):
+    """Strategy for design of experiments. This strategy is used to generate a set of
+    experiments for a given domain.
+    The experiments are generated via minimization of the D-optimality criterion.
+
+    """
+
+    def __init__(
+        self,
+        data_model: data_models.DoEStrategy,
+        **kwargs,
+    ):
+        super().__init__(data_model=data_model, **kwargs)
+        self.formula = data_model.formula
+        self.domain = data_model.domain
+
+    def _ask(self, candidate_count: PositiveInt) -> pd.DataFrame:
+        _fixed_experiments = self.candidates
+        _domain = self.domain
+        design = find_local_max_ipopt(
+            _domain,
+            self.formula,
+            n_experiments=candidate_count,
```

Comment:
```suggestion
            n_experiments=candidate_count+len(_fixed_experiments),
```

Reply:
why is this necessary again? The behaviour in the original doe package is the same: If i fix one experiments and request 2, I get two back with one being the one i fixed.